.github/SUPPORT.md

```diff
@@ -9,4 +9,4 @@ Welcome to Gutenberg, a WordPress project. We hope you join us in creating the f
 
 * Join us on Slack for real-time communication, it is where maintainers coordinate around the project. To get started using Slack, see: https://make.wordpress.org/chat/
 
-* For general WordPress support with the core editor, see the [WordPress.org support forums](https://wordpress.org/support/) — it is highly active and well maintained.
+* For general WordPress support with the core editor, see the [WordPress.org support forums](https://wordpress.org/support/forums/) — it is highly active and well maintained.
```

Comment:
This change is the only exception. This URL should be in the support forum, but accessing `https://wordpress.org/support/` redirects to `https://wordpress.org/documentation/`. Therefore, we have rewritten it to the appropriate link.